src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+    private final static int MIN_RANGE_NUMBER = 1;
+    private final static int MAX_RANGE_NUMBER = 45;
+    private final static String ERROR_MESSAGE = "로또 숫자는 1~45 사이이어야 합니다.";
+    private int value;
```

Comment:
상수와 인스턴스 변수 사이에는 한 줄 띄어서 가독성을 높여주세요! 🙏
```suggestion
    private final static String ERROR_MESSAGE = "로또 숫자는 1~45 사이이어야 합니다.";
    
    private int value;
```